.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,23 +1,25 @@
 <!--- Provide a general summary of your changes in the Title above -->
 
+## Intent of Change
+<!--- What types of changes does your code introduce?  -->
```

Comment:
I think we can remove that line. Would be nice to keep the comments to a minimum otherwise its a lot to remove each time